Makefile

```diff
@@ -448,10 +448,10 @@ deploy-dev-infra: create-hub-cluster
 ########
 
 test:
-	$(MAKE) _run_subsystem_test AUTH_TYPE=rhsso ENABLE_ORG_TENANCY=true ENABLE_ORG_BASED_FEATURE_GATES=true
+	$(MAKE) _run_subsystem_test AUTH_TYPE=rhsso ENABLE_ORG_TENANCY=true ENABLE_ORG_BASED_FEATURE_GATES=true TEST="$(or $(TEST),'github.com/openshift/assisted-service/subsystem')"
 
 test-kube-api:
-	$(MAKE) _run_subsystem_test AUTH_TYPE=local FOCUS="$(or ${FOCUS},kube-api)"
```

Comment:
No way this is running some code that is not local right ? e.g remote `github.com/openshift/assisted-service`